src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
Yes. NW.js allow you to run node code from the HTML on the client. That background page is a bridge between those worlds so it doesn't have to attach 2 debuggers one for node and other for chrome.